test/test_sparse_csr.py

```diff
@@ -2772,9 +2770,8 @@ def fn(input):
             dense_output.backward(dense_covector)
             self.assertEqual(sparse_input.grad, dense_input.grad)
 
-    @skipCUDAIfRocm
     @skipCUDAIf(
-        not _check_cusparse_sddmm_available(),
+        not (TEST_WITH_ROCM or _check_cusparse_sddmm_available()),
```

Comment:
Can the function - _check_cusparse_sddmm_available() updated to handle the TEST_WITH_ROCM condition.

Reply:
TEST_WITH_ROCM moved into _check_cusparse_sddmm_available() in the next commit